letsencrypt/storage.py

```diff
@@ -488,6 +499,18 @@ def ocsp_revoked(self, version=None):
         # certificate is not revoked).
         return False
 
+    def autorenewal_is_enabled(self):
```

Comment:
Coverage was coming from `test_should_autorenew`, but it doesn't hurt to have explicit unit tests.